lib/oban/worker.ex

```diff
@@ -2,33 +2,34 @@ defmodule Oban.Worker do
   @moduledoc """
   Defines a behavior and macro to guide the creation of worker modules.
 
-  Worker modules do the work of processing a job. At a minimum they must define a `perform/2`
-  function, which is called with an `args` map and the full `Oban.Job` struct.
+  Worker modules do the work of processing a job. At a minimum they must define
+  a `c:perform/1` function, which is called with the full `Oban.Job` struct.
 
   ## Defining Workers
 
-  Worker modules are defined by using `Oban.Worker`. A bare `use Oban.Worker` invocation sets a
-  worker with these defaults:
+  Worker modules are defined by using `Oban.Worker`. A bare `use Oban.Worker`
+  invocation sets a worker with these defaults:
 
   * `:max_attempts` — 20
   * `:priority` — 0
   * `:queue` — `:default`
-  * `:unique` — no uniquness set
+  * `:unique` — no uniqueness set
 
-  The following example defines a worker module to process jobs in the `events` queue. It then
-  dials down the priority from 0 to 1, limits retrying on failures to 10, adds a "business" tag,
-  and ensures that duplicate jobs aren't enqueued within a 30 second period:
+  The following example defines a worker module to process jobs in the `events`
+  queue. It then dials down the priority from 0 to 1, limits retrying on
+  failures to 10, adds a "business" tag, and ensures that duplicate jobs aren't
+  enqueued within a 30 second period:
 
       defmodule MyApp.Workers.Business do
         use Oban.Worker,
           queue: :events,
-          max_attempts: 10,
           priority: 1,
+          max_attempts: 10,
```

Comment:
I just changed the order of this option to follow the description given in the previous paragraph.